### PR TITLE
fix backend port wrong diff generated

### PR DESCRIPTION
**Pull request by cristian-radu**
- **What does the PR do?** *(JIRA reference, background context, references to other tickets)*
- **What automated testing exists for this change?** *Link to new or existing unit tests*
- **Is there documentation?** *If yes, please provide a link to the docs.prsn.io page or appropriate Readme*
- **Any dependencies?** *(PRs in other repos that need to be merged alongside this)*
- [x] **Has the changelog been updated?**

### CODE REVIEW

File: pkg/cluster/service_map.go [hunk 1]
```diff
@@ -166,7 +166,11 @@ func (s ServiceMap) AddIngress(ingress v1beta1_ext.Ingress) {
 	if backendService != biteservice.Name {
 		biteservice.Backend = backendService
 	}
-	biteservice.BackendPort = int(ingress.Spec.Rules[0].IngressRuleValue.HTTP.Paths[0].Backend.ServicePort.IntVal)
+	// backend port has been overriden
+	backendPort := int(ingress.Spec.Rules[0].IngressRuleValue.HTTP.Paths[0].Backend.ServicePort.IntVal)
```
Comment (geriBatai) [inline pkg/cluster/service_map.go]: doesn't this clash with multiple external_url feature you've added?

Comment (cristian-radu) [reply pkg/cluster/service_map.go]: maybe it's not obvious, but they are mutually exclusive.

**Comment by geriBatai** [comment]
LGTM

**Comment by geriBatai** [comment]
LGTM again

**Pull request merged**